.travis.yml

```diff
@@ -16,37 +16,53 @@ matrix:
     - php: '7.0'
     - php: '7.1'
     - php: '7.2'
-      env: coverage=1 lint=1
```

Comment:
I would skip those variables and the corresponding tests to speed up the builds.

Reply:
There is still the variable `coverage`, `lint` is not needed anymore as I'm defining the tests in the matrix instead of globally. IMO it's easier to maintain like this! See the `before_install` and `script` sections are only executed in this matrix. The other ones are using those defined globally below.